src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IConstructorBodyOperation.cs

```diff
@@ -122,18 +122,21 @@ public C() : base()
             compilation.VerifyOperationTree(node1, expectedOperationTree:
 @"
 IConstructorBodyOperation (OperationKind.ConstructorBody, Type: null) (Syntax: 'public C()  ... row null; }')
-  Initializer: 
+    Initializer: 
     IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null, IsImplicit) (Syntax: ': base()')
-      Expression: 
+        Expression: 
         IInvocationOperation ( System.Object..ctor()) (OperationKind.Invocation, Type: System.Void) (Syntax: ': base()')
-          Instance Receiver: 
+            Instance Receiver: 
             IInstanceReferenceOperation (ReferenceKind: ContainingTypeInstance) (OperationKind.InstanceReference, Type: System.Object, IsImplicit) (Syntax: ': base()')
-          Arguments(0)
-  BlockBody: 
+            Arguments(0)
+    BlockBody: 
     IBlockOperation (1 statements) (OperationKind.Block, Type: null) (Syntax: '{ throw null; }')
-      IThrowOperation (OperationKind.Throw, Type: null) (Syntax: 'throw null;')
-        ILiteralOperation (OperationKind.Literal, Type: null, Constant: null) (Syntax: 'null')
-  ExpressionBody: 
+        IThrowOperation (OperationKind.Throw, Type: null) (Syntax: 'throw null;')
+        IConversionOperation (TryCast: False, Unchecked) (OperationKind.Conversion, Type: System.Exception, Constant: null, IsImplicit) (Syntax: 'null')
```

Comment:
Formatting seems very weird here, operands should be indented from parents.. In general there are a bunch of formatting changes to the IOperation unit tests that don't seem relevant, please revert them.

Reply:
Unfortunately, there is no easy way to do that.  The new VS test explorer places an extra four spaces before the output of a test, so my choices are to either copy and paste that, adding the four spaces, or do the same followed by an "unindent" operation in VS, which unfortunately messes up the indentation as here.  If there was a way to see the raw output of the test I could use that, but I don't see that option.

---
In reply to: [301332549](https://github.com/dotnet/roslyn/pull/37052#discussion_r301332549) [](ancestors = 301332549)

Reply:
At the very least, the formatting needs to be consistent. The `IConversionOperation` here is a child of the `IThrowOperation` and needs to be formatted as such. (if you were using vsvim, fixing the spaces is as easy as selecting the lines and then `s/^    //g`).

---
In reply to: [301738225](https://github.com/dotnet/roslyn/pull/37052#discussion_r301738225) [](ancestors = 301738225,301332549)

Reply:
@jfleisher, I've filed feedback on this that was assigned to you: [https://developercommunity.visualstudio.com/content/problem/612756/test-explorer-adds-4-spaces-to-the-beginning-of-ev.html](https://developercommunity.visualstudio.com/content/problem/612756/test-explorer-adds-4-spaces-to-the-beginning-of-ev.html). This is the kind of scenario where the extra lines introduced by the test explorer hurts the compiler team.

---
In reply to: [301780343](https://github.com/dotnet/roslyn/pull/37052#discussion_r301780343) [](ancestors = 301780343,301738225,301332549)